packages/arb-token-bridge-ui/src/components/common/ProjectsListing.tsx

```diff
@@ -0,0 +1,108 @@
+import axios from 'axios'
+import useSWRImmutable from 'swr/immutable'
+import { ChevronRightIcon } from '@heroicons/react/24/outline'
+import { useNetworks } from '../../hooks/useNetworks'
+import { getNetworkName, isNetwork } from '../../util/networks'
+import { PortalProject, Project } from './PortalProject'
+import { PORTAL_API_ENDPOINT } from '../../constants'
+import { ExternalLink } from './ExternalLink'
+import { getBridgeUiConfigForChain } from '../../util/bridgeUiConfig'
+import { getChainQueryParamForChain } from '../../types/ChainQueryParam'
+import { trackEvent } from '../../util/AnalyticsUtils'
+
+const shuffleArray = (array: PortalProject[]) => {
+  return array.sort(() => Math.random() - 0.5)
+}
+
+const fetchProjects = async (chainId: number) => {
+  const isChainOrbit = isNetwork(chainId).isOrbitChain
+  const chainSlug = getChainQueryParamForChain(chainId)
+
+  if (!isChainOrbit || !chainSlug) {
+    return []
+  }
+
+  const response = await axios.get(
+    `${PORTAL_API_ENDPOINT}/api/projects?chains=${chainSlug}`
+  )
+
+  return response.data as PortalProject[]
+}
+
+export const ProjectsListing = () => {
+  const [{ destinationChain }] = useNetworks()
+
+  const isDestinationChainOrbit = isNetwork(destinationChain.id).isOrbitChain
+
+  const { color: destinationChainUIcolor } = getBridgeUiConfigForChain(
+    destinationChain.id
+  )
+
+  const destinationChainSlug = getChainQueryParamForChain(destinationChain.id)
+
+  const {
+    data: projects,
+    error,
+    isLoading
+  } = useSWRImmutable(['fetchProjects', destinationChain.id], () =>
+    fetchProjects(destinationChain.id)
+  )
+
+  if (
+    isLoading ||
+    !isDestinationChainOrbit ||
+    !projects ||
+    projects.length === 0 ||
+    typeof error !== 'undefined'
+  ) {
+    return null
+  }
```

Comment:
`!isDestinationChainOrbit` should be done at the key level

https://swr.vercel.app/docs/conditional-fetching

then you can probably use `isLoading || !projects` to return null for the component?